libs/runtime/ebpf_bitmap.h

```diff
@@ -101,4 +137,4 @@ extern "C"
 
 #ifdef __cplusplus
 }
-#endif
+#endif
```

Comment:
Why is this line changed?

Reply:
I am resolving this and other similar comments about the diff tool reporting changes when apparently no changes occurred, including white space.